### DISCUSSION TITLE
v3.3.1

### DISCUSSION THREAD

**Discussion by YiiGuxing** [YiiGuxing/TranslationPlugin#1332] [Versions]
## What's Changed

- 一些优化与改进
- 修复了谷歌文档翻译未知错误的问题
- 修复了Material Theme UI主题导致翻译对话框UI显示异常的问题
- 还修复了其他好多问题

<hr /><em>This discussion was created from the release <a href='https://github.com/YiiGuxing/TranslationPlugin/releases/tag/v3.3.1'>v3.3.1</a>.</em>

**Comment by RubyXun** [comment]
提出的问题得到了修复,👍

**Comment by Pandai20** [comment]
怎么设置新的快捷键？

**Reply by YiiGuxing** [reply]
> 怎么设置新的快捷键？

可以参考此文章：
https://zhuanlan.zhihu.com/p/97144428
